base_business_document_import/models/business_document_import.py

```diff
@@ -103,6 +104,19 @@ def _match_partner(
             partner_dict, chatter_msg, domain, partner_type_label)
         if partner:
             return partner
+        if partner_dict.get("gln"):
+            if "partner_identification_gln" in self.env.registry._init_modules:
+                # Only if partner_identification_gln is installed
+                # this code is executed
+                categ = self.env.ref(
```

Comment:
You can do this with `raise_if_not_found=False` and avoid the check  whether `partner_identification_gln` is installed. If `partner_identification_gln.partner_identification_gln_number_category` is found do the search otherwise skip.

Reply:
Thanks for your valuable review. I've have updated
if not installed, search() below will fail because of missing fields